### DISCUSSION TITLE
Allowing table row height to grow to fit text instead of truncating.

### DISCUSSION THREAD

**Discussion by brpaz** [directus/directus#4045] [Feature Requests]
Hello.

Right now the default behavior of Directus when the text from a cell is bigger than its size is to truncate the text. 

I have some very text oriented collections that I would like to display the full text on the list page, and to have the height of the table row grow to fit all the text. I know I can change the width of the columns but that might not always be enough.

Ex:

**Before:**
![1](https://user-images.githubusercontent.com/184563/107855065-1ffee100-6e18-11eb-8b24-86ea7804ccdf.png)

**After**
![2](https://user-images.githubusercontent.com/184563/107855066-20977780-6e18-11eb-8e98-fff936ae49d0.png)

I can achieve that with a custom CSS but I don´t think that there is any selector I can use to apply custom styles to a table for a specific collection for example.

So I would suggest having a way to apply custom CSS to a specific page (it could be as simple as Directus to add a css class with the current page to the body tag or more options to configure the table layout,

**Comment by benhaynes** [comment]
Two things:
1. Yes, we should definitely include helper classes/ids throughout the App for easier scoping of custom CSS.
2. I think having a core option for expanding row height on the Table Layout is a good idea! I think this should be a new option under `spacing` (see below) called "Content Wrap" (or something like that). If you have the code, you could submit a PR!

![Screen Shot 2021-02-15 at 10 38 51 AM](https://user-images.githubusercontent.com/522079/107966744-476dbf00-6f7a-11eb-8d19-65ced7837961.png)

**Comment by rijkvanzanten** [comment]
I don't know if we can without harming the functionality of displays elsewhere in the system 🤔 

Display extensions are currently designed and built to function as a single-line (so they work in all the various contexts they're used in), so we'd either have to make a multi-line preview display (and make sure it doesn't break all other things, like page title / card subtitle etc etc), or make a tabular layout specific setting to render the value as plain text inline (thus effectively skipping displays altogether)

**Comment by bamboowonder** [comment]
Just wondering if word-wrap cells in a table has had any traction in being implemented?   This would be exceptionally useful to us.

**Reply by rijkvanzanten** [reply]
> https://github.com/directus/directus/discussions/4045#discussioncomment-370943

This still applies..

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
